src/utils/utils.js

```diff
@@ -292,6 +295,10 @@ export const getconstructorParams = (abiConstructor, state, vals, crowdsaleNum)
     return params;
 }
 
+export function roundToDecimals(n, input) {
```

Comment:
1) Use an es6 arrow function not a javascript function
2) Since the return value can be held in one line I would re-write it as 
```
export const roundToDecimals = (n, input) => 1.0 / n * Math.ceil(n * input)
```